examples/meta/src/multiclass/multiclassmachine.sg

```diff
@@ -0,0 +1,33 @@
+File fm_train_real = csv_file("../../data/fm_train_real.dat")
+File fm_test_real = csv_file("../../data/fm_test_real.dat")
+File label_train_multiclass = csv_file("../../data/label_train_multiclass.dat")
+
+#![create_features]
+Features feats_train = features(fm_train_real)
+Features feats_test = features(fm_test_real)
+Labels labels_train = labels(label_train_multiclass)
+#![create_features]
+
+#![create_kernel]
+Kernel k = kernel("GaussianKernel", log_width=2.1)
+#![create_kernel]
+
+#![create_machine]
+Machine c = machine("LibSVM",epsilon=0.00005)
+#![create_machine]
+
+#![create_strategy]
+MulticlassStrategy multiclass_one_vs_rest_strategy = multiclass_strategy("KernelMulticlassMachine")
+#![create_strategy]
+
+#![create_classifier]
+MulticlassMachine mc_classifier = multiclass_machine("LibSVMOneClass", kernel=k,classifier=c,labels_train)
```

Comment:
I mean you need to look at what the interface examples look like and you'll see! Run `make meta_examples` and have a look at the output files, in your case the python file will be in something like `build/examples/meta/python/multiclass/multiclassmachine.py`

Reply:
look at the CI build error too: https://dev.azure.com/shogunml/shogun/_build/results?buildId=2676&view=logs&j=be3b03d7-4089-5ab3-9462-9af4d8b1dcdf&t=443ade06-68af-54ec-5bea-0e179d048f6f&l=93

Reply:
and try to run these things locally too so you can debug more quickly!